Package.swift

```diff
@@ -23,7 +23,7 @@ let package = Package(
         .target(
             name: "FluentUI",
             dependencies: [
-                .target(name: "FluentUI_ios", condition: .when(platforms: [.iOS, .visionOS])),
+              .target(name: "FluentUI_ios", condition: .when(platforms: [.iOS, .visionOS, .macCatalyst])),
```

Comment:
Could you fix the whitespace here? It should be the same as it was before the addition of `.macCatalyst`